notebooks/official/prediction/pytorch_train_deploy_models_with_prebuilt_containers.ipynb

```diff
@@ -0,0 +1,1400 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #34.                list : A list of dictionaries with predictions and explanations is returned
</pre><p>I would remove explanations because we did not configure the model or endpoint for explanations.</p><p> </p><p>Just to confirm, argmax works here because each output is an array of length 10, so in this case, the labels align with the array index. Is that right?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1728/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>

Reply:
<p>Agree. Removed explanation.</p>